libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ProbabilityDistributionBoxChart.tsx

```diff
@@ -4,6 +4,7 @@
 import { getTheme, IChoiceGroupOption } from "@fluentui/react";
 import {
   BasicHighChart,
+  boxChartTooltipDefaultSetting,
```

Comment:
where is this string defined?
Did we localize it?

Reply:
The existing tooltip isn't localized. I copied the default value and just replaced min/max with lower/upper whisker. I can certainly localize all the individual parts, though.